types/primus/index.d.ts

```diff
@@ -64,6 +64,9 @@ declare class Primus extends EventEmitter {
         done: (err: Error | null) => void,
     ): this;
 
+    connections: { [Key: string]: Primus.Spark };
```

Comment:
I would have called this `id` instead, but `Key` is fine also.

Reply:
Let me fix that and do a rebase <3